benches/reader.rs

```diff
@@ -0,0 +1,86 @@
+#![feature(test)]
+extern crate grcov;
+extern crate test;
+
+use grcov::{GcovReaderBuf, GCNO};
+use std::path::PathBuf;
+use test::{black_box, Bencher};
+
+#[bench]
+fn bench_reader_gcno(b: &mut Bencher) {
+    let mut gcno = GCNO::new();
+    b.iter(|| {
+        let file = GcovReaderBuf::from("test/llvm/reader.gcno");
```

Comment:
This is still benchmarking file reading too

Reply:
So, I based this off of the bench in [parser.rs](https://github.com/mozilla/grcov/blob/fd147e33b644b8d4422d07254f74462024541d20/benches/parser.rs#L14). I was getting the error `cannot move out of captured outer variable in an FnMut closure` outside of the iter and I cannot clone it either.